assets/python/matplotlib_basic.pyi

```diff
@@ -0,0 +1,90 @@
+# -------------------------------------------------------------------------------
+# This script contains a few helpful commands for basic plotting with matplotlib.
+# The commented out blocks are optional suggestions and included for convenience.
+# -------------------------------------------------------------------------------
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+import numpy as np
+
+
+# Plot Settings
+# -------------
+figsize        = (6.4, 4.8) # width, height [inches]
+dpi            = 100 # resolution [dots-per-inch]
+fsize_title    = 16 # font size of title
```

Comment:
Let's not use abbreviations where not established (e.g. "dpi"), please

figsize -> figure_size
fsize - font_size
fmt - format